src/internal_telemetry/allocations/allocator/token.rs

```diff
@@ -0,0 +1,146 @@
+use std::{
+    cell::RefCell,
+    num::NonZeroUsize,
+    sync::atomic::{AtomicUsize, Ordering},
+};
+
+use tracing::Span;
+
+use super::tracing::WithAllocationGroup;
+
+use crate::internal_telemetry::allocations::allocator::stack::GroupStack;
```

Comment:
Don't know what others think, but I prefer relative paths for related modules, as long as it doesn't end up with ridiculous numbers of `super::`:
```suggestion
use super::super::stack::GroupStack;
```